R/deployApp.R

```diff
@@ -729,15 +729,15 @@ runDeploymentHook <- function(appDir, option, verbose = FALSE) {
 }
 
 applicationDeleted <- function(client, deployment, recordPath, appMetadata) {
-  header <- "Failed to find existing application on server; it's probably been deleted."
+  header <- "Failed to find existing content on server; it's probably been deleted."
   not_interactive <- c(
     i = "Use {.fn forgetDeployment} to remove outdated record and try again.",
-    i = "Or use {.fn applications} to see other applications you have on the server."
+    i = "Or use {.fn applications} to see other content you have on the server."
   )
   prompt <- "What do you want to do?"
   choices <- c(
     "Give up and try again later",
-    "Delete existing deployment & create a new app"
+    "Delete existing deployment & create a new content"
```

Comment:
```suggestion
    "Delete existing deployment record & deploy this content as a new item"
```

The phrase "create a new content" isn't quite right. Feel free to adjust my suggestion, though.